client/Helpers/AccessEditor.elm

```diff
@@ -444,7 +444,11 @@ rowWrite config record =
 
           else
             none
-        , checkboxWrite Admin record
+        -- If record key not of type all but permission is alreay set, show checkbox for admin
+        , if not (record.key == All && not (hasPermission Admin record)) then
```

Comment:
```suggestion
        , if record.key /= All || (record|>hasPermission Admin) then
```
Better not to use double negative here.

Reply:
Right, fixed that. Thanks for pointing that out :)